dotnet/src/InternalUtilities/samples/InternalUtilities/BaseTest.cs

```diff
@@ -14,7 +14,7 @@ public abstract class BaseTest : TextWriter
     /// and <see cref="TestConfiguration.AzureOpenAI"/> are defined.
     /// If 'false', Azure takes precedence.
     /// </summary>
-    protected virtual bool ForceOpenAI { get; } = false;
+    protected virtual bool ForceOpenAI { get; } = true;
```

Comment:
nit: Do we need to commit this change?

Reply:
No that wasn't supposed to be included...backing out now.